source/clear-linux/virtual-environments/kvm.rst

```diff
@@ -1,57 +1,156 @@
-(Intel® VT-d) in your BIOS/UEFI firmware configuration.
+1. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
+   `Intel®Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
+   host machine’s BIOS.
 
-Starter script
-==============
+2. Log in and get root privilege on the host machine::
 
-To start the image, run the `qemu shell script`_ available in the
-`images`_ directory.
+    $ sudo -s
 
-SSH is needed for remote logins; however, SSH not enabled by default. To
-enable it, log in through serial console with the username ``root``. After
-setting the password, enable root login via SSH by configuring
-:file:`/etc/ssh/sshd_config` with this line::
+3. Install `QEMU-KVM` on the host machine. Below are some example distros.
 
-    PermitRootLogin yes
+   * |CL|::
 
-Now you may connect from host via SSH through 2223::
+     # swupd bundle-add desktop-autostart kvm-host
 
-    $ ssh -p 10022 root@localhost
+   * Ubuntu 16.04 LTS Desktop::
 
-Alternatively, there are a few other ways to approach this.
+     # apt-get install qemu-kvm
 
-*  To run the script without modifying its permissions::
+   * Mint 18.1 “Serena” Desktop::
 
-   $ bash start_qemu.sh clr_image
+     # apt-get install qemu-kvm
 
-*  To run it as a background process::
+   * Fedora 25 Workstation::
 
-   $ bash start_qemu.sh clr_image &
+     # dnf install qemu-kvm
 
-*  If you'd like to run the script with execute permission::
+4. Download the latest pre-built Clear Linux _`KVM image` file from
+   the _`image` directory.
 
-   $ chmod +x start_qemu.sh
-   $ ./start_qemu.sh clr_image
+5. Uncompress the downloaded image
 
-*  And to run it as a background process::
+   .. code-block:: console
 
-   $ ./start_qemu.sh clr_image &
+      unxz clear-<version number>-kvm.img.xz
 
-If you run into any trouble with qemu getting locked up, try editing the
-`qemu shell script`_ and removing the ``aio=threads``
+6. Download the :file:`OVMF.fd` file that provides UEFI support for
+   virtual machines from the _`image` directory.
 
+7. Download the sample _`QEMU-KVM launcher` script from the _`image` directory.
 
-.. _qemu shell script: http://download.clearlinux.org/image/start_qemu.sh
-.. _images: http://download.clearlinux.org/image/
+8. Make the script executable::
+
+   # chmod +x start_qemu.sh
+
+9. Start the |CL| KVM virtual machine::
+
+   # ./start_qemu.sh clear-<version number>-kvm.img
+
+#. Log in and set the root password.
+
+#. To SSH into the |CL| VM, follow these steps.
+
+    a. Enable SSH access
+
+       .. code-block:: console
+
+          # cat > /etc/ssh/sshd_config << EOF
+            PermitRootLogin yes
+            EOF
+
+    b. From the host, SSH into the Clear Linux VM::
+
+       # ssh -p 10022 root@localhost
+
+#. To add the GNOME Display Manager (GDM) to the |CL| VM, follow these steps:
+
+   a. Shutdown the active |CL| VM.
+
+   b. Install VNCViewer on the host machine.  Below are some example distros.
+
+      *  Clear Linux::
```

Comment:
Preface each distro with the preposition On. i.e. On Clear Linux, On Fedora 25 Workstation, etc.